src/Orc.FileAssociation/Services/FileAssociationService.cs

```diff
@@ -102,5 +117,26 @@ public async Task UndoAssociationFilesWithApplicationAsync(ApplicationInfo appli
             }
 
         }
+
+        public virtual async Task OpenPropertiesWindowForExtensionAsync(string extension)
+        {
+            var appPath = AppDomain.CurrentDomain.BaseDirectory;
+            var resourcesPath = Path.Combine(appPath, "Resources\\");
+            await OpenPropertiesWindowForExtensionAsync(extension, resourcesPath);
+        }
+
+        public virtual async Task OpenPropertiesWindowForExtensionAsync(string extension, string path)
+        {
+            var fileName = $"Click on 'Change' to select default {extension} handler.{extension}";
+            var finalPath = Path.Combine(path, fileName);
+            Log.Debug($"Creating a Resources directory");
```

Comment:
Empty lines between "logical blocks of code". This is really hard to read and understand.

Reply:
this logging can be removed, _directoryService.Create already logs this